server/pkg/storytelling/page.go

```diff
@@ -0,0 +1,248 @@
+package storytelling
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/reearth/reearth/server/pkg/builtin"
+	"github.com/reearth/reearth/server/pkg/property"
+)
+
+type Page struct {
+	id          PageID
+	property    PropertyID
+	title       string
+	swipe       bool
+	layers      LayerIDList
+	swipeLayers LayerIDList
+	blocks      BlockList
```

Comment:
storyId is required here, or a story has pages?

Reply:
yes the story has pages